### ISSUE TITLE
SC 2.2.1: 注記にて「状況」と「コンテキスト」で表記揺れがある

### ISSUE THREAD

**Issue by tsmd** [waic/wcag22#338]
[SC 2.2.1](https://waic.jp/translations/WCAG22/#timing-adjustable)の注記は現在、次のとおりの文章になっています。

> 注記
この達成基準は、制限時間の結果として、コンテンツ又は状況の予期せぬ変化を引き起こさないように利用者がタスクを完了できるようにするためのものである。この達成基準は、利用者の動作の結果としてのコンテンツ又はコンテキストの変化を制限する[達成基準 3.2.1](https://waic.jp/translations/WCAG22/#on-focus) と併せて考慮すること。

ここで、「コンテンツ又は状況の予期せぬ変化」という語が使われていますが、直後の文章では「コンテンツ又はコンテキストの変化」とあり、「状況」と「コンテキスト」とで訳語が揺れています。
どちらも、原文では「(unexpected) changes in content or context」です。
また、「change of context」は[用語集にもあるとおり](https://waic.jp/translations/WCAG22/#dfn-change-of-context)、「コンテキストの変化」と訳されていますので、「状況」ではなく「コンテキスト」で統一するべきではないかと思いました。

ご検討のほどよろしくおねがいします。

**Comment by momdo** [comment]
関連：https://github.com/waic/wcag21/issues/1417
解説書にある達成基準の訳も修正する必要。

**Comment by bakera** [comment]
WCAG 2.2本体の「状況」をざっとチェック:

<修正すべきもの>
2.2.1 注記

>この達成基準は、制限時間の結果として、コンテンツ又は*状況*の予期せぬ変化を引き起こさないように利用者がタスクを完了できるようにするためのものである。
>This success criterion helps ensure that users can complete tasks without unexpected changes in content or *context* that are a result of a time limit. 

→コンテキストに変更


用語「コンテキストの変化」 注記

>コンテンツの変化が、必ずコンテキストの変化になるとはかぎらない。アウトラインの展開、動的なメニュー、又はタブの切替などのコンテンツの変化は、それが上記のいずれか (例: フォーカス) を変更しないかぎり、*状況*を変化させるとは限らない。
>A change of content is not always a change of context. Changes in content, such as an expanding outline, dynamic menu, or a tab control do not necessarily change the *context*, unless they also change one of the above (e.g., focus).

→コンテキストに変更 (何でこれ堂々と残ってるんだ?)


<問題ないもの>
2.2.2 注記 4 

>コンテンツの読み込み中やそれに類似した状況の一部として表示されるアニメーションについては、この段階ですべての利用者に対していかなるインタラクションも発生する可能性がなく、かつコンテンツ読み込みの進行状況を表示しないことが利用者の混乱を招いたり、コンテンツが動作を停止した、又はコンテンツが破損しているという誤解を生じたりする可能性がある場合には、必要不可欠なものと考えることができる。
>An animation that occurs as part of a preload phase or similar situation can be considered essential if interaction cannot occur during that phase for all users and if not indicating progress could confuse users or cause them to think that content was frozen or broken.

→「状況」が二つあるがそれぞれ situation / progress の訳の一部

用語 緊急 (emergency)
>健康、安全、又は財産を守るために即座の行動を必要とする、突然で予期しない状況又は出来事。
>a sudden, unexpected situation or occurrence that requires immediate action to preserve health, safety, or property

→ situation の訳

WCAG 2 ガイダンスのレイヤー
>様々な利用者層や状況からくるニーズを満たすために、三つの適合レベルが定義されている: A (最低レベル)、AA、AAA (最高レベル)。
> In order to meet the needs of different groups and different situations, three levels of conformance are defined: A (lowest), AA, and AAA (highest). 

→ situation の訳
